### PR TITLE
fix, refactor, chore(egen): adds numpy==1.23.0 and replaces google-vizier==0.0.4 to resolve dependency errors, replaces REGION with LOCATION, refactors code

### PR DESCRIPTION
**Pull request by sumanvita-springml**
This PR includes the following changes:

1. Adds numpy==1.23.0 installation and replaces google-vizier==0.0.4 to resolve dependency errors
2. changes REGION to LOCATION
3. Includes Colab restart Kernal and authentication as per template guidelines
4. Adds collab enterprise logo
5. Makes structural changes according to the new notebook template
6. Removes the use of future tense
7. Contraction of words

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/vizier/gapic-vizier-multi-objective-optimization.ipynb [hunk 1]
```diff
@@ -61,9 +63,9 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial demonstrates [Vertex AI Vizier](https://cloud.google.com/vertex-ai/docs/vizier/overview) multi-objective optimization. Multi-objective optimization is concerned with mathematical optimization problems involving more than one objective function to be optimized simultaneously.\n",
+        "This tutorial demonstrates Vertex AI Vizier multi-objective optimization. Multi-objective optimization is concerned with mathematical optimization problems involving more than one objective function to be optimized simultaneously.\n",
         "\n",
-        "Learn more about [Vertex AI Vizier](https://cloud.google.com/vertex-ai/docs/vizier/overview)."
+        "Learn more about [Vertex AI Vizier overview](https://cloud.google.com/vertex-ai/docs/vizier/overview)."
```
Comment (kittyabs) [inline notebooks/official/vizier/gapic-vizier-multi-objective-optimization.ipynb:68]: remove "overview"

**Review by kittyabs — COMMENTED** [review]
1 small edit

**Review by kittyabs — COMMENTED** [review]
no edits needed

**Review by katiemn — APPROVED** [review]

**Pull request merged**